KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -219,7 +219,7 @@ public enum UITexts {
 
     /* Booking */
     public enum Booking {
-        public static let bookNow = "Text.Booking.BookNow".localized
+        public static let next = "kh_uisdk_next".localized
```

Comment:
@mionescu95 can you please confirm you've used the same key for this on your cleanup?

Reply:
I found this key in Android ui sdk